### DISCUSSION TITLE
Inline chunk evaluation in VScode

### DISCUSSION THREAD

**Discussion by abduazizR** [quarto-dev/quarto-cli#8123] [Feature Requests]
### Description

Hi

I wonder if there is a way to show inline evaluation of code the same way in Rstudio. See the image below
![image](https://github.com/quarto-dev/quarto-cli/assets/61837330/040aedae-8c8c-4ffa-acd5-cf14d29de63e)

I have searched on many websites and could not do the same thing in Quarto in VScode while I can do it in Quarto in Rstudio. Is this a bug on my end, or is the feature not yet supported?

**Comment by mcanouil** [comment]
Feature requests and bug reports regarding the Quarto extension for VSCode are to be made on the following repository: https://github.com/quarto-dev/quarto
